RedlinesProject/ControlCard.xaml.cs

```diff
@@ -22,64 +22,65 @@ namespace RedlinesProject
+            "FocusDisengaged",
+        };
+
+        private List<string> indeterminateStates = new List<string>
+        {
+            "IndeterminateNormal",
+            "IndeterminatePointerOver",
+            "IndeterminatePressed",
+            "IndeterminateDisabled",
+        };
+
         public ControlCard(Type controlType)
         {
             this.InitializeComponent();
-
-            CreateLightThemeControlStates(controlType);
-            CreateDarkThemeControlStates(controlType);
+            
+            CreateControlStates(controlType);
         }
 
-        private void CreateDarkThemeControlStates(Type ctrlType)
+        private ControlStateViewer CreateControlStateViewer(Type cType, List<string> states)
         {
-            StateViewerDarkTheme.ControlType = ctrlType;
-            StateViewerDarkTheme.States = new List<string>
-            {
-                "Normal",
-                "PointerOver",
-                "Pressed",
-                "Disabled",
-                "UncheckedNormal",
-                "UncheckedPointerOver",
-                "UncheckedPressed",
-                "CheckedNormal",
-                "CheckedPointerOver",
-                "CheckedPressed",
-                "CheckedDisabled",
-                "IndeterminateNormal",
-                "IndeterminatePointerOver",
-                "IndeterminatePressed",
-                "IndeterminateDisabled",
-                "FocusEngagedHorizontal",
-                "FocusEngagedVertical",
-                "FocusDisengaged",
-            };
+            ControlStateViewer ctrlStateViewer = new ControlStateViewer();
+            ctrlStateViewer.ControlType = cType;
+            ctrlStateViewer.States = states;
+
+            return ctrlStateViewer;
         }
 
-        private void CreateLightThemeControlStates(Type ctrlType)
+        private void CreateControlStates(Type ctrlType)
         {
-            StateViewerLightTheme.ControlType = ctrlType;
-            StateViewerLightTheme.States = new List<string>
+            LightControlContainer_UnselectedGrid.Children.Add(CreateControlStateViewer(ctrlType, unselectedStates));
+
+            if (ctrlType == typeof(CheckBox))
             {
-                "Normal",
-                "PointerOver",
-                "Pressed",
-                "Disabled",
-                "UncheckedNormal",
-                "UncheckedPointerOver",
-                "UncheckedPressed",
-                "CheckedNormal",
-                "CheckedPointerOver",
-                "CheckedPressed",
-                "CheckedDisabled",
-                "IndeterminateNormal",
-                "IndeterminatePointerOver",
-                "IndeterminatePressed",
-                "IndeterminateDisabled",
-                "FocusEngagedHorizontal",
-                "FocusEngagedVertical",
-                "FocusDisengaged",
-            };
+                LightSelectedLabel.Visibility = Visibility.Visible;
```

Comment:
Should we collapse `LightUnselectedLabel` by default too? If unselected is the only state group we probably don't need to show the label?  Also, we should probably remove the "light" prefix? Looks like this is used by dark theme too.

Reply:
Oh whoops, these weren't needed anymore. :) good catch!